starter-code/models/celebrity.js

```diff
@@ -0,0 +1,11 @@
+const mongoose= require('mongoose')
+
+let CelebritySchema = new mongoose.Schema({
+    name: String,
+    occupation: String,
+    catchPhrase: String,
+}) 
+
+let celebrityModel = mongoose.model('celebritie', CelebritySchema)
+
+module.exports=celebrityModel
```

Comment:
Its best practice to add some spaces in these cases, for readability.
`module.exports = celebrityModel`